src/components/navBar/navBar.js

```diff
@@ -434,6 +439,21 @@ function MdNavItem($$rAF) {
           scope.$apply();
         });
       });
+
+      var stopDisabledWatch = angular.noop;
+
+      if (angular.isDefined(attrs.disabled)) {
+        mdNavItem.disabled = true;
+      }
+      else if (attrs.ngDisabled) {
+        stopDisabledWatch = scope.$parent.$watch(attrs.ngDisabled, function() {
+          mdNavItem.disabled = $parse(attrs.ngDisabled)(scope.$parent);
+        });
+      }
```

Comment:
I thought about something similar to the autocomplete

```js
$attrs.$observe('disabled', function (value) {
  mdNavItem.disabled= $mdUtil.parseAttributeBoolean(value, false); 
});
```

This should be everything you need to do.